apps/meteor/tests/e2e/login.spec.ts

```diff
@@ -50,4 +50,10 @@ test.describe.parallel('Login', () => {
 			await expect(poUtils.mainContent).toBeVisible();
 		});
 	});
+
+	test('Should correctly display switch language button', async ({ page }) => {
+		test.use({ locale: 'pt-BR' });
+
+		await expect(page.getByText('Change to Default')).toBeVisible();
```

Comment:
instead of asserting the button by text, why not testing the button behavior and checking if the language changed, and doing the query by the button name? so its a win-win

Reply:
I could have indeed tested clicking the button, but I didn't because... It didn't cross my mind :melting_face:

Reply:
<img width='320px' src='https://media1.tenor.com/m/ukA1njjTacAAAAAd/empty-brain.gif' />

Reply:
fixed